--- /usr/src/linux-3.0-rc4/include/linux/socket.h	2011-06-21 03:25:46.000000000 +0000
+++ /usr/src/linux-3.0-rc4-ipv6_ext/include/linux/socket.h	2011-06-25 23:21:02.000000000 +0000
@@ -192,7 +192,8 @@
 #define AF_IEEE802154	36	/* IEEE802154 sockets		*/
 #define AF_CAIF		37	/* CAIF sockets			*/
 #define AF_ALG		38	/* Algorithm sockets		*/
-#define AF_MAX		39	/* For now.. */
+#define AF_INET6_EXT    39
+#define AF_MAX		40	/* For now.. */
 
 /* Protocol families, same as address families. */
 #define PF_UNSPEC	AF_UNSPEC
@@ -234,6 +235,7 @@
 #define PF_IEEE802154	AF_IEEE802154
 #define PF_CAIF		AF_CAIF
 #define PF_ALG		AF_ALG
+#define PF_INET6_EXT    AF_INET6_EXT
 #define PF_MAX		AF_MAX
 
 /* Maximum queue length specifiable by listen.  */
